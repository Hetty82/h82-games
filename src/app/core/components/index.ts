import { AppComponent } from './app/app.component'
import { LayoutComponent } from './layout/layout.component'
import { NotFoundComponent } from './not-found/not-found.component'

export const components: any[] = [
  AppComponent,
  LayoutComponent,
  NotFoundComponent,
]

export * from './app/app.component'
export * from './layout/layout.component'
export * from './not-found/not-found.component'
