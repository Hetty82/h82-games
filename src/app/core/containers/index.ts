import { AppComponent } from './app/app.component'
import { GamesComponent } from './games/games.component'
import { LoginComponent } from './login/login.component'
import { NotFoundComponent } from './not-found/not-found.component'

export const containers: any[] = [
  AppComponent,
  GamesComponent,
  LoginComponent,
  NotFoundComponent,
]

export * from './app/app.component'
export * from './games/games.component'
export * from './login/login.component'
export * from './not-found/not-found.component'
