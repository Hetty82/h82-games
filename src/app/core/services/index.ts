import { GamesService } from './games.service'
import { UserService } from './user.service'

export const services: any[] = [
  GamesService,
  UserService,
]

export * from './games.service'
export * from './user.service'
