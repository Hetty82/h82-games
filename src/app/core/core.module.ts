import { NgModule } from '@angular/core'
import { CommonModule } from '@angular/common'
import { RouterModule } from '@angular/router'

import * as fromComponents from './components'
import * as fromContainers from './containers'
import * as fromServices from './services'


@NgModule({
  imports: [
    CommonModule,
    RouterModule,
  ],
  declarations: [
    ...fromComponents.components,
    ...fromContainers.containers,
  ],
  exports: [
    fromContainers.AppComponent,
  ],
})
export class CoreModule {
  static forRoot() {
    return {
      ngModule: CoreModule,
      providers: fromServices.services,
    }
  }
}
