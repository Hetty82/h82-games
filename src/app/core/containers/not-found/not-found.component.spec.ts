import { TestBed, async } from '@angular/core/testing'
import { NotFoundComponent } from './not-found.component'

describe('NotFoundComponent', () => {
  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [
        NotFoundComponent
      ],
    }).compileComponents()
  }))
  it('should create the component', async(() => {
    const fixture = TestBed.createComponent(NotFoundComponent)
    const component = fixture.debugElement.componentInstance
    expect(component).toBeTruthy()
  }))
})
