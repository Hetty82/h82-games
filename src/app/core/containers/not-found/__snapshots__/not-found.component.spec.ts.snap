// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundComponent should match snapshot 1`] = `
<app-not-found>
  <p>
     Oops, page not found..
  
  </p>
</app-not-found>
`;
